### ISSUE TITLE
Android always autoPlay

### ISSUE THREAD

**Issue by sulagarde** [up-inside/react-native-youtube-sdk#35]
Hi,

I encountered a problem with Android: It seems like is entering autoPlay even if it set to false.
Everything works perfect on iOs

Here is implementation:
```javascript
<YouTubePlayer
    videoId={videoId}
    fullscreen
    startTime={5}
    style={{ width: '100%', height: isTalk ? 150 : 200 }}
    onReady={(e) => console.log('onReady', e)}
    onError={(e) => console.log('onError', e.error)}
    onChangeState={(e) => console.log('onChangeState', e.state)}
    onChangeFullscreen={(e) => console.log('onChangeFullscreen', e.isFullscreen)}
/>
```

versions:
react-native-youtube-sdk: 1.0.4
react-native: 0.60.6

**Comment by s-aleinik** [comment]
Have the same issue. Android always autoplay, iOS is fine

android: 9
react-native: 0.61.5
react-native-youtube-sdk: 1.0.6

**Comment by srfaytkn** [comment]
we fixed the bug, use version 1.0.8 @s-aleinik @sulagarde
